library/src/main/java/com/timehop/stickyheadersrecyclerview/HeaderPositionCalculator.java

```diff
@@ -19,6 +19,9 @@
   private final OrientationProvider mOrientationProvider;
   private final HeaderProvider mHeaderProvider;
   private final DimensionCalculator mDimensionCalculator;
+  
+  private final Rect bufferRect1 = new Rect();
+  private final Rect bufferRect2 = new Rect();
```

Comment:
Does something like `mTempRect2` sound better? I can't think of any much better name...